### PR TITLE
Possible fix of Gender issue

### PR DESCRIPTION
**Pull request by Romfrosk**
## Description
Possible fix for issue #54 
Pokemon were Male Gloom before scout, Manual Scout made it a Female and the Icon for gender and pokémon has also changed

## How Has This Been Tested?
Tested on latest commit of the MIX_MEWTWO Branch
Running on Debian and using PG Scout

## Screenshots (if appropriate):
Original Male Before: gloom before
Original Male After: gloom after
Also affects both genders

Original Female Before: female gloom before
Original Female After: female gloom after

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Comment by Romfrosk** [comment]
Also looks like this is happening both on weather boosted and normal pokémon

File: pogom/models.py [hunk 1]
```diff
@@ -2294,6 +2295,7 @@ def parse_map(args, map_dict, scan_coords, scan_location, db_update_queue,
                     'move_2': pokemon_info.move_2,
                     'height': pokemon_info.height_m,
                     'weight': pokemon_info.weight_kg,
+                    'gender': pokemon_info.gender,
```
Comment (rlodev67) [inline pogom/models.py]: Did you test it without PGScout ?
I think you have to write
`'gender': pokemon_info.pokemon_display.gender,`

Comment (SkOODaT) [reply pogom/models.py]: your incorrect rlodev67 it is just pokemon_info.gender....BUT p.pokemon_display.gender needs to be removed above in models its now randomized below 30 just like IVs so gender needs to be scouted or encountered

Comment (rlodev67) [reply pogom/models.py]: It should be `'gender': pokemon_info.pokemon_display.gender`
Confirmed by stock RM

**Comment by Romfrosk** [comment]
Changed as requested. again :)

**Comment by rlodev67** [comment]
thanks

**Pull request merged**